### DISCUSSION TITLE
pytvlwcharts V.0.0.1

### DISCUSSION THREAD

**Discussion by MooneDrJune** [TechfaneTechnologies/pytvlwcharts#3] [Q&A]
# Python Tradingview's Lightweight-Charts (pytvlwcharts)
An Experimental Python Wrapper For [Tradingview's Lightweight-Charts](https://tradingview.github.io/lightweight-charts/) To Be Used In Notebook Environments. [Example Lightweight-Charts](https://codesandbox.io/examples/package/lightweight-charts) 

## Instructions on running the program for the first time
Install The `pytvlwcharts` Library Via `pip install -U git+https://github.com/TechfaneTechnologies/pytvlwcharts.git` And Then Open A Jupyter Notebook Locally or On [Google Colab](https://colab.research.google.com/)

Please Refer [pytvlwcharts.ipynb](https://github.com/TechfaneTechnologies/pytvlwcharts/blob/main/pytvlwcharts.ipynb) for usage details.

<hr /><em>This discussion was created from the release <a href='https://github.com/TechfaneTechnologies/pytvlwcharts/releases/tag/v.0.0.1'>pytvlwcharts V.0.0.1</a>.</em>

**Comment by Rahib4** [comment]
How do I create a line chart using this library, I am checking the line chart code.. what I dont understand is the format of straddle_df in the sample codes. when I simply select day close and pass it as straddle_df.. line chart comes blank.

**Comment by 0xAskar** [comment]
when i import this with pip3 install in a normal python library, the IDE says no such module exists when I try to import

**Reply by MooneDrJune** [reply]
It is supposed to be used in a notebook environment

**Reply by 0xAskar** [reply]
Ah I see, do you know of any other python lightweight-charts library, or see yourself developing for a python env outside of notebooks?

**Comment by abhinav-d3v** [comment]
Hey, can you please share some documentation on adding annotations on the graph,  Thanks

**Comment by xuenliang2019** [comment]
Hello, I can generate charts on google colab, but when I run same codes in notebook locally, the output is blank, and also no error message. So can you please tell me anything I should do except run "pip install --quiet --upgrade git+https://github.com/TechfaneTechnologies/pytvlwcharts.git".
